backend/test/test_server/unit/compute/test_diffexp_cxg.py

```diff
@@ -2,6 +2,8 @@
 
 import numpy as np
 
+
```

Comment:
the comments, filename and code in this file are misleading.  They refer to "CXG", but that data format is no longer supported by the `server` backend.   The code loads an H5AD, but claims it is testing the CXG adaptor.  Please rename the file, correct the comments, etc, to refer to H5AD.